src/json.hpp

```diff
@@ -7079,14 +8108,36 @@ class serializer
     void dump_float(number_float_t x)
     {
         // NaN / inf
-        if (not std::isfinite(x) or std::isnan(x))
```

Comment:
Why can `NaN` be ignored here?

Reply:
Its actually not ignored: the `isnan` test is just redundant. `isfinite` return false for NaN